components/chart/index.tsx

```diff
@@ -175,7 +240,7 @@ function Chart({
             changeZoomLevel={changeZoomLevel}
           />
         </div>
-        < DescriptionDrawer module={selectedModule} open={!!selectedModule} onClose={() => setSelectedModule(undefined)}>
+        < DescriptionDrawer updateURLWithNodeID={updateURLWithNodeID} module={selectedModule} open={!!selectedModule} onClose={() => setSelectedModule(undefined)}>
```

Comment:
```suggestion
        <DescriptionDrawer module={selectedModule} open={!!selectedModule} onClose={() => setSelectedModule(undefined)}>
```
Don't keep empty space before the component name, that might cause problems when building the project